src/main/resources/jenkins/branch/BranchSource/config.jelly

```diff
@@ -24,7 +24,7 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:branch="/lib/branch-api">
   <branch:rowWrapper name="source">
-    <branch:entryWrapper>
+    <branch:entryWrapper colspan="3">
```

Comment:
Should this not be a part of `entryWrapper.jelly`?

Reply:
The `entryWrapper` tag defines `colspan` as an optional attribute because the tag is used in a few places, and not all of them want `colspan="3"`.